### PR TITLE
added a refresh token example

### PR DESCRIPTION
**Pull request by wangxiaoxuan273**
Signed-off-by: wangxiaoxuan273 <wangxiaoxuan119@gmail.com>

Following pr #182 
A patch for issue #149 

Moved clientConfigExample to the front, and added refreshTokenExample at the end.

### CODE REVIEW

File: registry/remote/auth/example_test.go [hunk 1]
```diff
@@ -22,26 +22,27 @@ import (
...
 )
 
 const (
-	username    = "test_user"
-	password    = "test_password"
-	accessToken = "test/access/token"
+	username     = "test_user"
+	password     = "test_password"
+	accessToken  = "test/access/token"
+	refreshToken = "test/access/refreshToken"
```
Comment (Wwwsylvia) [inline registry/remote/auth/example_test.go:35]: nit:
```suggestion
	refreshToken = "test/refresh/token"
```

**Comment by Wwwsylvia** [comment]
cc @qweeah

File: registry/remote/auth/example_test.go [hunk 2]
```diff
@@ -199,41 +265,23 @@ func ExampleClient_Do_withAccessToken() {
 	// 200
 }
 
-// ExampleClient_Do_clientConfiguration shows the client configurations available,
-// including using cache, setting user agent and configuring OAuth2.
-func ExampleClient_Do_clientConfiguration() {
+func ExampleClient_Do_withRefreshToken() {
```
Comment (qweeah) [inline registry/remote/auth/example_test.go:268]: Doc: we still need commentary doc for this func

File: registry/remote/auth/example_test.go (hunk 1 above)
Comment (wangxiaoxuan273) [reply registry/remote/auth/example_test.go:35]: changed accordingly.

File: registry/remote/auth/example_test.go (hunk 2 above)
Comment (wangxiaoxuan273) [reply registry/remote/auth/example_test.go:268]: Added the commentary doc.

**Review by Wwwsylvia — APPROVED** [review]
LGTM

**Pull request merged**